package.json

```diff
@@ -13,16 +13,26 @@
     "@chakra-ui/react": "^2.3.7",
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
+    "@reduxjs/toolkit": "^1.9.0",
     "@types/node": "18.11.9",
     "@types/react": "18.0.25",
     "@types/react-dom": "18.0.8",
+    "chakra-ui-carousel": "^1.0.6",
     "eslint": "8.27.0",
     "eslint-config-next": "13.0.2",
     "framer-motion": "^7.6.5",
     "next": "13.0.2",
+    "nuka-carousel": "^5.4.0",
```

Comment:
if you are not using this please remove it 

to remove npm package you can use `npm uninstall {packageName}` in this case its `npm uninstall nuka-carousel`

docs: https://docs.npmjs.com/cli/v8/commands/npm-uninstall/

Reply:
Ohh my bad gonna remove it rn